src/site/stages/build/plugins/check-cms-urls.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-console */
+const ignoredPages = new Set(['drupal/test/index.html']);
+
+function checkForCMSUrls(BUILD_OPTIONS) {
+  return (files, metalsmith, done) => {
+    const filesWithBadUrls = [];
+    for (const fileName of Object.keys(files)) {
+      const file = files[fileName];
+      if (file.isDrupalPage && !ignoredPages.has(fileName)) {
+        const contents = file.contents.toString();
+        if (
+          contents.includes('cms.va.gov') ||
+          contents.includes('va.agile6.com')
+        ) {
+          filesWithBadUrls.push(fileName);
+        }
+      }
+    }
+
+    if (filesWithBadUrls.length) {
+      console.log(
+        'The following pages have cms.va.gov or va.agile6.com referenced:',
+      );
```

Comment:
By launch, do you think this should fail the build?

Reply:
It'll actually fail the build now, the error is thrown a couple lines down.

Reply:
Ah yes there it is lol 🤦‍♂️